### PR TITLE
fix(client): preload next page before flip

### PR DESCRIPTION
**Pull request by zhpadvoiski**
### :pencil: Description
- show blurred background with loading comparadise logo while images from the next page are loading.
- removing unnecessary data refetching

https://github.com/ExpediaGroup/comparadise/assets/7442958/0969d238-3279-4d3c-bdfd-0c819f5b8612

### :link: Related Issues
-

### CODE REVIEW

File: app/frontend/components/main-page.tsx [hunk 1]
```diff
@@ -33,66 +47,56 @@ export const MainPage = () => {
...
   if (nextPageExists) {
     utils.fetchCurrentPage.prefetch({ hash, bucket, page: page + 1 });
   }
 
-  useEffect(() => {
-    if (data) {
-      getViewType(data.images).then(newViewType => setViewType(newViewType));
+  React.useEffect(() => {
+    if (!isNextPageReady && data?.images) {
+      preloadNextPage(data.images).then(newViewType => {
```
Comment (zhpadvoiski) [inline app/frontend/components/main-page.tsx:69]: preloading images from the next page

File: app/frontend/cypress/component/App.cy.tsx [hunk 2]
```diff
@@ -77,7 +78,7 @@ describe('App', () => {
       cy.findByRole('button', { name: /side-by-side/i }).should('be.disabled');
     });
 
-    it('should disable arrow buttons while loading next spec', () => {
+    it.skip('should disable arrow buttons while loading next spec', () => {
```
Comment (zhpadvoiski) [inline app/frontend/cypress/component/App.cy.tsx:81]: skipping as this functionality does not exist anymore. 
we disable arrow only when we currently on the first or the last page

Comment (danadajian) [reply app/frontend/cypress/component/App.cy.tsx:81]: Let's delete this test then!

**Review by danadajian — APPROVED** [review]

**Pull request merged**